ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -143,7 +149,7 @@ public struct Avatar: View, TokenizedControlView, Equatable {
         // Adding ringInnerGapOffset to ringInnerGap & ringThickness to accommodate for a small space between
         // the ring and avatar when the ring is visible and there is no inner ring gap
         let ringInnerGapOffset = 0.5
-        let ringInnerGap: CGFloat = isRingVisible ? (hasRingInnerGap ? tokenSet[.ringInnerGap].float : -ringInnerGapOffset) : 0
+        let ringInnerGap: CGFloat = (isRingVisible || hasBackgroundOutline) ? ((hasRingInnerGap || hasBackgroundOutline) ? tokenSet[.ringInnerGap].float : -ringInnerGapOffset) : 0
```

Comment:
nit: is there a way to make this easier to read? maybe multiple lines like we have below?

Reply:
Yeah I have another, bigger refactor in the wings that completely refactored this logic, but I agree that we should still try to keep it legible in the interim.

Reply:
Update: I've switch to using the outer gap for this API, which I think fits better with the intent anyway (and the values are identical). Plus, it makes for cleaner code!

Reply:
looks good, ship it!